card/src/main/res/values/styles.xml

```diff
@@ -94,4 +93,16 @@
         <item name="android:layout_gravity">center_vertical</item>
     </style>
 
+    <style name="AdyenCheckout.Card.AddressLookup.DropdownTextInputLayout" parent="AdyenCheckout.DropdownTextInputLayout">
+        <item name="endIconMode">none</item>
+    </style>
+
+    <style name="AdyenCheckout.Card.AddressLookup.DropdownTextInputEditText" parent="AdyenCheckout.DropdownTextInputEditText">
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_marginBottom">@dimen/standard_margin</item>
+        <item name="android:minHeight">@dimen/input_height</item>
+        <item name="android:hint">Search Address</item>
+        <item name="android:text">@string/checkout_address_lookup_hint</item>
```

Comment:
The color of the end drawable doesn't change color in dark mode. We can fix this with setting it's color to the text color I guess 
```XML
<item name="drawableTint">?android:attr/textColor</item>
```

Reply:
Very very good eye on detail.

Reply:
fixed